flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/util/serialization/JSONDeserializationSchema.java

```diff
@@ -29,6 +30,7 @@
  *
  * <p>Fields can be accessed by calling objectNode.get(&lt;name>).as(&lt;type>)
  */
+@Internal
```

Comment:
This could also be `@PublicEvolving`, I don't think there's a problem with the user using it directly.

Reply:
The reason I started to do this is because I think the user is less likely to use this method.